packages/tools/client-debugger/devtools-view/package.json

```diff
@@ -74,6 +74,7 @@
 		"@fluidframework/sequence": "workspace:~",
 		"@fluidframework/test-runtime-utils": "workspace:~",
 		"@fluidframework/tinylicious-client": "workspace:~",
+		"@fluentui/react-icons-mdl2": "~1.3.41",
```

Comment:
If this is being used in anything other than test code, it needs to be added to the `dependencies` list, rather than `devDependencies`.

Reply:
Also, it would be worth checking if `@fluentui/react-icons` has what you need, since we already have that dependency.

Reply:
I'd also recommend moving to @fluentui/react-icons. Not sure why there's two, but last I checked we were more on this one, and mdl2 feels more "internal" to fluentui.

Reply:
Removed mdl2 here. Updated.